package-lock.json

```diff
@@ -1,8 +1,8148 @@
 {
```

Comment:
Could we revert the change of this file? We should not change this in a swagger PR.

Reply:
Done